cookbook/assetic/uglifyjs.rst

```diff
@@ -73,8 +73,8 @@ your JavaScripts:
         assetic:
             filters:
                 uglifyjs2:
-                    # the path to the uglifyjs executable
-                    bin: /usr/local/bin/uglifyjs
+                    # the path to the uglifyjs Node module
+                    bin: ./app/Resources/node_modules/uglify-js/bin/uglifyjs
```

Comment:
Well not !! Mine is at default place (no prefix option) but it was to keep consistency with local install explain previously in this doc.